docs/README.md

```diff
@@ -1,10 +1,5 @@
 # Contour Documentation
 
-The contents of this directory have moved to [projectcontour.io](https://projectcontour.io/).
-Specifically;
+The contents of this directory have moved to corresponding directories in [site/content](site/content).
```

Comment:
Since this file is `docs/README.md`  the link tries to open `docs/site/content` so should the link rather be `../site/content`?

Reply:
You're right, it should be that. Because this page is rendered at https://github.com/projectcontour/contour/tree/main/docs, I thought it would be relative to the `main` directory, but GitHub rewrites the link to be absolute, built as a relative link from the `docs` directory. I will update.